src/bmark/bmark_solvers.jl

```diff
@@ -37,20 +38,28 @@ if Pkg.installed("BenchmarkProfiles") != nothing
   * `stats`: a dict of statistics such as obtained from `bmark_solvers()`
 
   #### Keyword arguments
-  Any keyword argument accepted by `BenchmarkProfiles.performance_profile()`.
+  * `cost`: a function `cost(::ExecutionStats)` returning a positive cost value. Usually, time or number of function evaluations.
```

Comment:
It combines for the profile, but not for stopping conditions. Our discussion started around the question: `max_eval` stopping condition should look only at `neval_obj`, all counters, or a subset of counters?